deploy/cluster-monitoring-config-non-uwm/config.yaml

```diff
@@ -1,10 +1,12 @@
 deploymentMode: "SelectorSyncSet"
 selectorSyncSet:
-  matchLabelsApplyMode: "OR"
```

Comment:
I moved the 4.5 specific out of here so we don't have the `matchLabelsApplyMode: "OR"`